components/classes/consentpolicy.schema.json

```diff
@@ -0,0 +1,70 @@
+{
+  "meta:license": [
+    "Copyright 2025 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/classes/consentpolicy",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Consent Policy",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "description": "This is the base class for a consent policy definition which includes fields like name, description, version, and status. This class can be used to build Consent Policy XDM schemas.",
+  "meta:extends": [
+    "https://ns.adobe.com/xdm/data/record",
+    "https://ns.adobe.com/xdm/common/auditable"
+  ],
+  "definitions": {
+    "consentpolicy": {
+      "properties": {
+        "xdm:consentPolicyName": {

```

Comment:
"xdm:name" - This is a member of the consentpolicy object, so the longer name is redundant.